parts/linux/cloud-init/artifacts/components.json

```diff
@@ -282,16 +282,16 @@
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/livenessprobe:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v2.12.0",
-        "v2.13.0"
+        "v2.13.0",
+        "v2.13.1"
       ]
     },
     {
       "downloadURL": "mcr.microsoft.com/oss/kubernetes-csi/csi-node-driver-registrar:*",
       "amd64OnlyVersions": [],
       "multiArchVersions": [
-        "v2.10.1",
```

Comment:
pls keep existing sidecar images until they are not supported in deprecated aks versions, I need to add those deleted sidecar images back